06week/higherOrder.js

```diff
@@ -2,24 +2,47 @@
 
 const assert = require('assert');
 
-function forEach(arr, callback) {
-  // Your code here
+const forEach = (arr, callback) => {
+  for(let i = 0 ; i < arr.length ; i++){
+    callback(arr[i],i,arr)
```

Comment:
unless your using the extra arguments . You should only pass one index of the array at a time.